src/mavlink_vehicles.hh

```diff
@@ -42,39 +45,66 @@ struct state_variable {
     }
 };
 
+//!
+//! @brief Store vehicle attitude in radians
+//!
 struct attitude : state_variable {
     float roll;
     float pitch;
     float yaw;
 };
 
+//!
+//! @brief Store vehicle global position in ENU
+//!
 struct global_pos_int : state_variable {
-    int32_t lat; // Degrees * 1e7
-    int32_t lon; // Degrees * 1e7
-    int32_t alt; // Millimeters (AMSL)
+    int32_t lat; //! @brief Latitude in degrees * 1e7
+    int32_t lon; //! @brief Longitude in degrees * 1e7
+    int32_t alt; //! @brief Millimeters (AMSL)
```

Comment:
There is something odd happening here:

![screenshot from 2016-12-07 12-41-12](https://cloud.githubusercontent.com/assets/1584229/20985742/96895e44-bc7a-11e6-9898-656df4c19d77.png)